README.md

```diff
@@ -60,18 +60,23 @@ Code completion, highlighting, browsing and validation for High Level Assembler
 [![GitHub issues](https://img.shields.io/github/issues-raw/zowe/vscode-extension-for-zowe?style=flat-square)](https://github.com/zowe/vscode-extension-for-zowe/issues)
 [![slack](https://img.shields.io/badge/chat-on%20Slack-blue?style=flat-square)](https://openmainframeproject.slack.com/)
 
-Zowe Explorer is a VS Code extension powered by Zowe CLI that streamlines interaction with mainframe data sets, USS files, and jobs.
+Zowe Explorer is a VS Code extension powered by Zowe CLI that streamlines interaction with mainframe data sets, USS files, and jobs. The extension is designed to function along with other extensions and plug-ins to deliver a richer experience.
+
+You can learn more about the Zowe Explorer by watching the [Getting Started](https://www.youtube.com/embed/G_WCsFZIWt4) and [Work with Data Sets](https://www.youtube.com/embed/X4oSHrI4oN4) tutorial videos.
 
 > How can we improve Zowe Explorer? [Let us know on our Git repository](https://github.com/zowe/vscode-extension-for-zowe/issues)
 
 ### Features
-* Access z/OS Datasets and z/OS Unix file systems and submit JCLs.
+* Access z/OS Datasets and z/OS Unix file systems, and submit JCLs.
+* Create, edit, and work with z/OSMF compatible profiles.
+* Store your credentials securely with Secure Credentials Store plug-in.
 * View and download job output.
 * Issue TSO commands.
+* Install additional extensions.

```

Comment:
What exactly is meant by this?

Reply:
This is to describe the potential of Zowe Explorer. the fact that it has extensions for itself. For example, we have the SCS plug-in for it now. I anticipate more plug-ins/extensions for ZE in the future.